tests/unit/common.hpp

```diff
@@ -126,8 +126,10 @@ ::testing::AssertionResult seq_almost_eq(Seq1&& seq1, Seq2&& seq2) {
     for (std::size_t j = 0; i1!=e1 && i2!=e2; ++i1, ++i2, ++j) {
         using FP = testing::internal::FloatingPoint<FPType>;
 
-        auto v1 = *i1;
-        auto v2 = *i2;
+        // cast to FPType to avoid warnings about lowering conversion
+        // if FPType has lower precision than Seq{12}::value_type
+        auto v1 = FPType(*i1);
+        auto v2 = FPType(*i2);
```

Comment:
I think the warning was actually providing us a good service; using the `float` test to compare `double`-valued sequences was a quick hack to get around not having a `::testing::seq_near` function. Doing a 4 ulp float test on doubles is violating the semantics implicitly promised by the name (analogous to the gtest names), so we should live with the warning until we implement the correct test.